services/src/configuration/src/main/java/org/openkilda/config/KafkaTopicsConfig.java

```diff
@@ -93,12 +93,13 @@
     @Default("kilda.topo.disco")
     String getTopoDiscoTopic();
 
-    @Key("topo.cache")
-    @Default("kilda.topo.cache")
-    String getTopoCacheTopic();
-
     @Key("topo.nbworker")
     @FallbackKey("kafka.nbworker.topic")
     @Default("kilda.topo.nb")
     String getTopoNbTopic();
+
+    @Key("topo.reroute")
+    @FallbackKey("kafka.reroute.topic")
```

Comment:
Why do you need "fallback" name for NEW option?

Reply:
Fixed.